### ISSUE TITLE
Create a page that describes performance/resource related concepts and FAQ

### ISSUE THREAD

**Issue by lmatz** [risingwavelabs/risingwave-docs-legacy#1201]
https://www.notion.so/risingwave-labs/What-is-the-right-cluster-configuration-for-Risingwave-a93d162114b1450ebc569e922afca56d?pvs=4

Intend to position it as a documentation page instead of a blog post because the content on this page needs to evolve.

~Still a WIP draft and I will collect more opinions about the content. I gotta go through it multiple times before publishing I suppose...~

I have gone through it multiple times.

**Comment by neverchanje** [comment]
https://github.com/risingwavelabs/risingwave-docs/issues/1193 The internal concepts introduction would be necessary before the tuning doc.

**Comment by lmatz** [comment]
> #1193 The internal concepts introduction would be necessary before the tuning doc.

I agree, and the concepts of actors/fragments are also the pre-requisite of introducing the concept of `parallelism`.
For example, setting an MV to N `parallelism` does not mean that it will use N CPUs(multiplied by the number of CNs) at most. Upstream and downstream fragments are parallel with each other too.

**Comment by lmatz** [comment]
Also depends on the doc of `scaling in/out` as we want users to adjust resources when they switch from the phase of reading historical data to the phase of reading new data.

link #857
